tools/pythonpkg/src/python_import_cache.cpp

```diff
@@ -17,7 +17,11 @@ bool PythonImportCacheItem::LoadSucceeded() const {
 
 bool PythonImportCacheItem::IsLoaded() const {
 	auto type = (*this)();
-	return type.ptr() != nullptr;
+	bool loaded = type.ptr() != nullptr;
```

Comment:
Not sure what the point of this change is?

Reply:
None really, I had initially meant to check ModuleIsLoaded if IsLoaded returned true, to double verify
But even with monkeypatching this looked like it didn't make ModuleIsLoaded return false

So this is left ovee code that I might want to undo